### PR TITLE
Install loopback CNI plugin when using amazon vpc cni

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

**Comment by olemarkus** [comment]
similar to #9286, but different plugin

File: nodeup/pkg/model/networking/amazonvpc.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+package networking
+
+import (
+	"k8s.io/kops/nodeup/pkg/model"
+
+	"k8s.io/kops/upup/pkg/fi"
+)
+
+// FlannelBuilder writes Flannel's assets
```
Comment (MoShitrit) [inline nodeup/pkg/model/networking/amazonvpc.go:25]: Should that comment be updated to `// AmazonVPCBuilder writes Amazon VPC's assets`?

Comment (olemarkus) [reply nodeup/pkg/model/networking/amazonvpc.go]: Thanks for spotting

**Comment by hakman** [comment]
@olemarkus @rifelpet how do you feel about adding back the `NetworkBuilder` and always extract *all* CNI binaries to `/opt/cni/bin`?

**Comment by olemarkus** [comment]
The loopback plugin is fine, maybe on the standard ones, I'd prefer to avoid those that are nearly never used (like flannel)

**Comment by hakman** [comment]
Sounds good for me. I guess we will add flannel just for Flannel, Calico and Canal.

**Comment by olemarkus** [comment]
Superseded by #9310 
/close

**Pull request closed without merging**